app/assets/stylesheets/components/form/_form.scss

```diff
@@ -5,3 +5,18 @@
 .form__input-container--medium {
   max-width: 140px;
 }
+
+%form-row-2up {
```

Comment:
We avoided designing forms like this as we have other components that'll clash with it like this https://www.moneyadviceservice.org.uk/en/tools/house-buying/mortgage-affordability-calculator